packages/core/src/providers/chainState/multiChainStates/provider.tsx

```diff
@@ -81,9 +81,7 @@ export function MultiChainStateProvider({ children, multicallAddresses }: Props)
     }
 
     const callsOnThisChain = uniqueCalls.filter((call) => call.chainId === chainId)
-    if (callsOnThisChain.length === 0) {
```

Comment:
Why do we remove this?

Reply:
i put that on multicall function, to reduce duplicated code